### ISSUE TITLE
miniavc-v2.dll keeps disappearing on startup #403

### ISSUE THREAD

**Issue by OddSobriquet** [ihsoft/KIS#405]
As mentioned in #403 modulemanager crashes fatally on startup.

KSP log
```
[EXC 16:41:24.753] [ModuleManager] The patching thread threw an exception: System.IO.FileNotFoundException: C:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\GameData\KIS\Plugins\MiniAVC-V2.dll
  at System.Diagnostics.FileVersionInfo.GetVersionInfo (System.String fileName) [0x0000d] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
  at ModuleManager.ModListGenerator.GenerateModList (System.Collections.Generic.IEnumerable`1[T] modsAddedByAssemblies, ModuleManager.Progress.IPatchProgress progress, ModuleManager.Logging.IBasicLogger logger) [0x000a2] in <ac37db06097f4706b70066f555f86981>:0 
  at ModuleManager.MMPatchLoader.Run () [0x000f3] in <ac37db06097f4706b70066f555f86981>:0 
  at ModuleManager.MMPatchRunner+<>c__DisplayClass10_0.<Run>b__1 () [0x00001] in <ac37db06097f4706b70066f555f86981>:0 
  at ModuleManager.Threading.BackgroundTask+<>c__DisplayClass0_0.<Start>g__RunAction|0 () [0x00002] in <ac37db06097f4706b70066f555f86981>:0
```

re-installing KIS brings back miniavc-v2.dll
starting KSP deletes it --> modulemanager crashes fatally
renaming miniavc-v2.dll.pruned to miniavc-v2.dll and restarting ksp deletes dll again, modulemanager crashes...

See attached video for clarification

https://user-images.githubusercontent.com/72888738/200879021-7a7c0b2f-a135-41f5-89c4-ff272760b486.mp4

edit: KSP 1.12.4
Module Manager 4.2.2

**Comment by ihsoft** [comment]
KIS doesn't delete or rename anything. It seems you have a mod like [ZeroMiniAVC](https://spacedock.info/mod/1614/ZeroMiniAVC) which does it. Please, share the log.

**Comment by OddSobriquet** [comment]
Well I'll be damned and I do apologize sincerely; I do feel fairly embarrassed, right now....

I have (had) zero miniavc installed. I tried uninstalling through ckan before opening the ticket, but ckan told me 4 other mods had it in their dependencies, thus I didn't follow through.... I have manually taken the mod out of the game data folder now and so far there seem to be no issues....
I will head over to the mods who claim they need to have it installed and probably enquire what's going on there...
It's just weird as before the update to 12.4 it still worked (kinda) and before 12.3 it worked most certainly as I haven't had a change to my setup in literal ages. After I had modulemanager crash, I reinstalled it and all was dandy....

If you wanna sift through the logs regardless, knock yourselves out, at this point, I don't think it's necessary, though.
[KSP.log](https://github.com/ihsoft/KIS/files/9978172/KSP.log)

my apologies again, should've done better testing before opening the ticket.
